packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -1,10 +1,22 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, { MouseEventHandler, HTMLProps } from 'react';
 import Icon from '../icon';
 import Text from '../text';
 
-const Checkbox = React.forwardRef(
+type TCheckBoxProps = Omit<HTMLProps<HTMLInputElement>, 'value'> & {
+    className: string;
+    classNameLabel: string;
+    defaultChecked: boolean;
+    disabled: boolean;
+    greyDisabled: boolean;
+    id: string;
+    label: string; //or object : packages/p2p/src/components/order-details/order-details-confirm-modal.jsx
```

Comment:
```suggestion
    label: string;
```
@iman-fs Localize returns a string, so no comment needed, label's just a string🙂